enterprise/releases/2023-06-0.mdx

```diff
@@ -0,0 +1,77 @@
+   Amazon RDS/Aurora database server
+  - In case you have installed the collector separately, we recommend upgrading the collector package/container and reviewing the [full changelog](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md#0500------2023-06-05)
+- Issue e-mail notifications: Include pganalyze organization name in email body, for easier filtering when using multiple organizations
+- Add additional telemetry data to check pganalyze snapshot health via Prometheus/Grafana
+  - See [instructions](/docs/enterprise/settings#worker-queue-monitoring-using-prometheus) for how to use the Prometheus endpoint
+- Allow setting `SIDEKIQ_CONCURRENCY=0` to turn off workers in combined Enterprise image
+  - You can utilize this setting in order to separate the workers from the container servicing the pganalyze web UI, avoiding slow web requests caused by worker processing
+  - Note such a setup requires having configured object storage as well as an external Redis server (set via `REDIS_URL`)
+  - See [all settings](/docs/enterprise/settings)
+
+
+## Performance
+
+- Query analysis: Speed up analysis by avoiding to process schema dependency tracking
+- Improve performance of getIssues by avoiding N+1 queries
+- Disable use of prepared statements for web application
+  - In practice this ends up causing more overhead than benefit for the application, and we've seen better results without prepared statements
+- Cache table names associated to each query directly on `queries` table
+  - This speeds up background tasks working on a per-table basis (e.g. Index Advisor), as well as per-table queries list in the web application
+- Background workers:
+  - Don't run check-up logic for servers that have not received data in the last 24 hours
+  - Speed up truncation of queries (truncated queries are used for display in tables)
+  - Batch check-up logic together for 10 database servers at a time
+
+
+## Bugfixes
+
+- Object storage: Fix DNS lookup errors from Rust-based snapshot workers
+  - This may have shown as an I/O error that caused no snapshot data to be processed, despite the collector successfully sending data to the installation
+  - If we have previously asked you to utilize an IP address for your MinIO endpoint as a workaround, you can now switch back to using a DNS hostname
+- Query analysis:
+  - Don't retry query analysis after 'relation does not exist' errors
+  - Avoid "Unexpected rtekind" errors for functions and VALUES
+  - Fix planning for hash indexes (this allows Index Advisor to run on tables that have hash indexes, but does not yet recommend hash indexes)
+- Remove query length limit for parsing
+  - Previously query texts exceeding 20,000 characters were not parsed during the initial intake, causing them to not have referenced tables, or an associated statement type. This is change removes this limit.
```

Comment:
```suggestion
  - Previously query texts exceeding 20,000 characters were not parsed during the initial intake, causing them to not have referenced tables, or an associated statement type. This is change removes this limit
```

Looks like this one only ends with period, let's remove it for consistency

Reply:
Yep, good catch - will remove that last sentence and the trailing dot.